templates/deployments.yaml

```diff
@@ -70,6 +70,13 @@ spec:
         # TODO: Add liveness/readiness probes (#3)
         #livenessProbe:
         #readinessProbe:
+        {{- if .Values.st2auth.env }}
+        env:
+        {{- range $env, $value := .Values.st2auth.env }}
+        - name: {{ $env | quote }}
+          value: {{ $value | quote }}
+        {{- end }}
+        {{- end }}
```

Comment:
I think this could be extracted into a helper to avoid some repetition.
So, here it would have:
```
{{ include "custom-env" .Values.st2auth | indent 8 }}
```
And then the template would be:
```
{{- define "custom-env" -}}
  {{- if .env }}
env:
    {{- range $env, $value := .env }}
- name: {{ $env | quote }}
  value: {{ $value | quote }}
    {{- end }}
  {{- end }}
{{- end -}}
```